procedures/unit-testing-basics.ipf

```diff
@@ -1459,6 +1463,17 @@ static Function TestCaseBegin(testCase)
 
 	WAVE/T wvSuite = UTF_Reporting#GetTestSuiteWave()
 	UTF_Reporting#UpdateChildRange(wvSuite, wvTestCase)
+	wvSuite[%current][%NUM_TESTS] = num2istr(str2num(wvSuite[%current][%NUM_TESTS]) + 1)
+
+	WAVE/T wvTestRun = UTF_Reporting#GetTestRunWave()
+	wvTestRun[%current][%NUM_TESTS] = num2istr(str2num(wvTestRun[%current][%NUM_TESTS]) + 1)
```

Comment:
They are counting something similar but the current row in `wvSuite` is changed more often than in `wvTestRun`. I can also refactor it so that `wvTestRun` is only updated after `wvSuite` is finished with counting.

Reply:
I moved the update for test run to TestSuiteEnd